Loosely backported from upstream commit cbf61e1588048fe655b8228407fbb10973da7b5f.
Author: Jan Synacek <jsynacek@redhat.com>

The original upstream commit changed the error message in the first hunk.
The hunk is removed in this patch instead, because it is a dead code.
The device variable is always initialized to "eth0".

--- iputils-s20071127/arping.c	2016-09-12 09:44:07.777681697 +0200
+++ iputils-s20071127-new/arping.c	2016-09-12 09:45:42.837615491 +0200
@@ -465,11 +465,6 @@ main(int argc, char **argv)
 
 	target = *argv;
 
-	if (device == NULL) {
-		fprintf(stderr, "arping: device (option -I) is required\n");
-		usage();
-	}
-
 	if (s < 0) {
 		errno = socket_errno;
 		perror("arping: socket");
--- iputils-s20071127/doc/arping.sgml	2016-09-12 09:44:07.777681697 +0200
+++ iputils-s20071127-new/doc/arping.sgml	2016-09-12 09:44:31.254665346 +0200
@@ -19,7 +19,7 @@
 <arg choice="opt">-c <replaceable/count/</arg>
 <arg choice="opt">-w <replaceable/deadline/</arg>
 <arg choice="opt">-s <replaceable/source/</arg>
-<arg choice="req">-I <replaceable/interface/</arg>
+<arg choice="opt">-I <replaceable/interface/</arg>
 <arg choice="req"><replaceable/destination/</arg>
 </cmdsynopsis>
 </refsynopsisdiv>
@@ -82,8 +82,7 @@ Finish after the first reply confirming
  <varlistentry>
   <term><option><anchor id="opt.interface">-I <replaceable/interface/</option></term>
   <listitem><para>
-Name of network device where to send ARP REQUEST packets. This option
-is required.
+Name of network device where to send ARP REQUEST packets. Default is eth0.
   </para></listitem>
  </varlistentry>
